### PR TITLE
Load /etc/environment file to runner

### PR DESCRIPTION
**Pull request by yhaliaw**
Load the /etc/environment file with contents from a jinja template.

### CODE REVIEW

File: .github/workflows/end_to_end_test.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ jobs:
     steps:
       - name: Echo hello world
         run: echo "hello world"
+      - name: Proxy set in /etc/environment
```
Comment (jdkandersson) [inline .github/workflows/end_to_end_test.yaml:15]: What about using microk8s? E.g., installing it?

File: templates/environment.j2 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
+HTTP_PROXY={{proxies['http']}}
+HTTPS_PROXY={{proxies['https']}}
+NO_PROXY={{proxies['no_proxy']}}
+http_proxy={{proxies['http']}}
+https_proxy={{proxies['https']}}
+no_proxy={{proxies['no_proxy']}}
```
Comment (jdkandersson) [inline templates/environment.j2:7]: Final new line?

File: .github/workflows/end_to_end_test.yaml (hunk 1 above)
Comment (jdkandersson) [reply .github/workflows/end_to_end_test.yaml:15]: And deploy maybe a hello world container and check it works?

File: .github/workflows/end_to_end_test.yaml [hunk 3]
```diff
@@ -12,6 +12,14 @@ jobs:
...
+        run: cat /etc/environment | grep PROXY
+      - name: Install microk8s
+        run: sudo snap install microk8s --classic
+      - name: Wait for microk8s
+        run: sudo microk8s status --wait-ready
+      - name: Deploy nginx for testing
+        run: sudo microk8s kubectl create deployment nginx --image=nginx
```
Comment (jdkandersson) [inline .github/workflows/end_to_end_test.yaml:22]: Should we also include a check that the deployment actually worked? Like a curl or something?

Comment (weiiwang01) [inline .github/workflows/end_to_end_test.yaml:22]: Should we check that nginx is actually deployed? For example `kubectl rollout status deployment/nginx --timeout=5m`

Comment (yhaliaw) [reply .github/workflows/end_to_end_test.yaml:22]: Good idea

**Review by jdkandersson — DISMISSED** [review]

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Pull request merged**